file_store/src/traits/msg_verify.rs

```diff
@@ -40,29 +32,35 @@ impl_msg_verify!(SpeedtestReqV1, signature);
 impl_msg_verify!(LoraBeaconReportReqV1, signature);
 impl_msg_verify!(LoraWitnessReportReqV1, signature);
 impl_msg_verify!(DataTransferSessionReqV1, signature);
-impl_msg_verify!(OrgCreateHeliumReqV1, signature);
-impl_msg_verify!(OrgCreateRoamerReqV1, signature);
-impl_msg_verify!(OrgDisableReqV1, signature);
-impl_msg_verify!(OrgEnableReqV1, signature);
-impl_msg_verify!(RouteStreamReqV1, signature);
-impl_msg_verify!(RouteListReqV1, signature);
-impl_msg_verify!(RouteGetReqV1, signature);
-impl_msg_verify!(RouteCreateReqV1, signature);
-impl_msg_verify!(RouteUpdateReqV1, signature);
-impl_msg_verify!(RouteDeleteReqV1, signature);
-impl_msg_verify!(RouteGetEuisReqV1, signature);
-impl_msg_verify!(RouteUpdateEuisReqV1, signature);
-impl_msg_verify!(RouteGetDevaddrRangesReqV1, signature);
-impl_msg_verify!(RouteUpdateDevaddrRangesReqV1, signature);
-impl_msg_verify!(GatewayLocationReqV1, signature);
-impl_msg_verify!(GatewayRegionParamsReqV1, signature);
-impl_msg_verify!(AdminAddKeyReqV1, signature);
-impl_msg_verify!(AdminLoadRegionReqV1, signature);
-impl_msg_verify!(AdminRemoveKeyReqV1, signature);
-impl_msg_verify!(SessionKeyFilterGetReqV1, signature);
-impl_msg_verify!(SessionKeyFilterListReqV1, signature);
-impl_msg_verify!(SessionKeyFilterStreamReqV1, signature);
-impl_msg_verify!(SessionKeyFilterUpdateReqV1, signature);
+impl_msg_verify!(iot_config::OrgCreateHeliumReqV1, signature);
```

Comment:
i've started pathing these message types here since they will have very similar names between other services (like the mobile config service)